src/js/features/progress.js

```diff
@@ -312,7 +346,9 @@ Object.assign(MediaElementPlayer.prototype, {
 		media.addEventListener('timeupdate', (e) => {
 			if (media.duration !== Infinity ) {
 				player.setProgressRail(e);
-				player.setCurrentRail(e);
+				if(!t.forcedHandlePause){
```

Comment:
yeah it is necesary when `media.setCurrentTime(t.newTime);` is not added to handleMouseMove function